public/src/game/GameSettings.jsx

```diff
@@ -57,9 +57,10 @@ const SortCards = () => (
 );
 
 const sizeDisplay = {
-  "Small": "Low",
-  "Normal": "Medium",
-  "Large": "High",
+  "text": "Text",
```

Comment:
`Text-Only`, `No Image` or simply `None`?

There are not all not quite accurate though with the option label being `Card image quality:`... but better than `Text`?